### PR TITLE
Add Dependabot for ingestion API

### PR DESCRIPTION
**Pull request by dhruvkb**
## Description

The label check are failing on Dependabot PRs due to inadequate labelling. This PR fixes that.

## Technical details

It also configures Dependabot to check the ingestion-server for updates.

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by AetherUnbound — APPROVED** [review]
Nice!

File: .github/dependabot.yml [hunk 1]
```diff
@@ -6,19 +6,40 @@
...
     # Look for a `Pipfile` in the `/openverse-api` directory
     directory: '/openverse-api'
     # Check for updates once a week
     schedule:
       interval: 'weekly'
+    labels:
+      - "💻 aspect: code"
+      - "🧰 goal: internal improvement"
+      - "dependencies"
+      - "python"
```
Comment (sarayourfriend) [inline .github/dependabot.yml:32]: What's the python label for, just curious?

**Review by sarayourfriend — APPROVED** [review]
LGTM

File: .github/dependabot.yml (hunk 1 above)
Comment (dhruvkb) [reply .github/dependabot.yml:32]: It's just the label Dependabot used to add for Pip updates to distinguish the language (helpful when using in a repo with multiple languages). It's not a label that we've defined.

Comment (sarayourfriend) [reply .github/dependabot.yml:32]: Oh gotcha! I didn't understand why we would want it given the entire repo is Python but if it's not ours to decide then there you go 🙂

**Pull request merged**